.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.18-openshift-4.12
+  tag: rhel-8-release-golang-1.19-openshift-4.12
```

Comment:
rhel-8-release-golang-1.19-openshift-4.13 would be the correct one here and everywhere else too.

Reply:
done